elasticapm/instrumentation/packages/elasticsearch.py

```diff
@@ -30,18 +30,15 @@
 
 from __future__ import absolute_import
 
-import json
+import re
 
 import elasticapm
 from elasticapm.instrumentation.packages.base import AbstractInstrumentedModule
-from elasticapm.utils import compat
 from elasticapm.utils.logging import get_logger
 
 logger = get_logger("elasticapm.instrument")
 
-
-API_METHOD_KEY_NAME = "__elastic_apm_api_method_name"
-BODY_REF_NAME = "__elastic_apm_body_ref"
+should_capture_body_re = re.compile("_((search|msearch)(/template)?|count)$")
```

Comment:
FWIW, this is what the node.js agent is currently using:

```
// URL paths matching the following pattern will have their query params and
// request body captured in the span (as `context.db.statement`). We match
// a complete URL path component to attempt to avoid accidental matches of
// user data, like `GET /my_index_search/...`.
const pathIsAQuery = /\/(_search|_msearch|_count|_async_search|_sql|_eql)(\/|$)/
```

That adds `_async_search` (requested during some Elasticsearch perf work by @ gingerwizard), and a couple others. It also matches full URL segments to avoid accidental matches.

Reply:
I realize I haven't yet opened anything on apm.git to spec this out.

Reply:
@trentm awesome, thanks! I actually copied the regex from the Node.js agent, but from the original PR that introduced the regex, not from current master :facepalm:

Reply:
A shared spec would be great for this!